From 89d4aece7ba679703060393ac95086fd514c7fc7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Tue, 27 Mar 2018 19:09:44 +0200
Subject: [PATCH 2/2] fix implicit declaration warnings by including string.h
 and unistd.h

string.h is needed for strncmp(3) and unistd.h is needed for read(3),
close(3) and lseek(3).
---
 posixtz.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/posixtz.c b/posixtz.c
index 972ca31..4a36e10 100644
--- a/posixtz-0.5/posixtz.c
+++ b/posixtz-0.5/posixtz.c
@@ -14,6 +14,8 @@
 #include <fcntl.h>
 #include <limits.h>
 #include <stdio.h>
+#include <string.h>
+#include <unistd.h>
 
 #include "posixtz.h"
 
-- 
2.16.2

